xds/xds.go

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package xds contains xds implementation. Users need to import this package to
```

Comment:
Is there some canonical documentation we can link to about what xDS is and how to get started?

Reply:
Not yet. Our doc doesn't have instruction to setup the control plane. The wallet example would be a good place, but it doesn't exist.

Reply:
Will we be adding something to http://grpc.io?  Let's try to remember to come back and add a pointer when we have one.

Reply:
Maybe a link to envoy for now?  As a user that doesn't know what xds is, this message means nothing to me: "xds contains xds. Import this to get xds"  :smile:

Reply:
I linked to the xds example. At least that (tried to) explain what xDS is. And we would update the example to point to whatever new doc we will have.

Reply:
This is fine for now.  We should definitely have a link to our "official" xds documentation from this package's godoc once it exists.

Reply:
We tried this same thing when writing the xds example, and failed to find a good link to post there... We will need our official doc.

Reply:
How about the xDS gRFC?

Reply:
The example readme does link to the gRFC.